CryptoTechReminderSystem/DomainObject/SlackDeveloper.cs

```diff
@@ -3,5 +3,6 @@ namespace CryptoTechReminderSystem.DomainObject
     public class SlackDeveloper : Developer
     {
         public string Id { get; set; }
+        public string Email { get; set; }
```

Comment:
Why has email been added to the `SlackDeveloper` domain object- isn't this already inherited from the `Developer` domain object?

Reply:
Yes, you're correct. I've resolved this in https://github.com/chubberlisk/crypto_tech/pull/12/commits/241c5127ab03baeb4b25393506f2c84c5ec0b706.